src/index.ts

```diff
@@ -53,6 +67,43 @@ export const beautifier: Beautifier = {
   },
 };
 
+function findFile({
+  finishPath = "/",
+  startPath = finishPath,
+  fileNames,
+}: {
+  startPath: string | undefined;
+  finishPath: string | undefined;
+  fileNames: string[];
+}): Promise<string> {
+  // Recursive case: Check & go uuuuppp
+  const filePaths = fileNames.map(fileName => path.join(startPath, fileName));
+  // tslint:disable-next-line
+  return Promise.all(filePaths.map(filePath => doesFileExist(filePath)))
+  .then(exists => filePaths.filter((filePath, index) => exists[index]))
+  .then(foundFilePaths => {
+    if (foundFilePaths.length > 0) {
+        return foundFilePaths[0];
+    }
+    const parentDir = path.resolve(startPath, "../");
```

Comment:
This line with `parentDir` can be moved to after the `if` statement, since it is not used yet.